samples/apps/github-qna-webapp-react/README.md

```diff
@@ -0,0 +1,41 @@
+# GitHub Repo Q&A Bot Sample
+
+> **!IMPORTANT**
+> This learning sample is for educational purposes only and should not be used in any
+> production use case. It is intended to highlight concepts of Semantic Kernel and not
+> any architectural / security design practices to be used.
+
+## Running the sample
+
+1. You will need an [Open AI Key](https://openai.com/api/) or
+   [Azure Open AI Service key](https://learn.microsoft.com/azure/cognitive-services/openai/quickstart)
+   for this sample.
+2. Ensure the service API is already running `http://localhost:7071`. If not learn
+   how to start it [here](../../dotnet/KernelHttpServer/README.md).
+3. **Run** the following command `yarn install` (if you have never run the sample before)
+   and/or `yarn start` from the command line.
+4. A browser will open or you can navigate to `http://localhost:3000` to use the sample.
+
+## About the GitHub Repo Q&A Bot Sample
+
+The GitHub Repo Q&A Bot sample allows you to pull in data from a public GitHub repo
+into a local memory store in order to ask questions about the project and to get
+answers about it.  The sample highlights how [memory](https://aka.ms/sk/memories)
+and [embeddings](https://aka.ms/sk/embeddings) work along with SK Functions when
+the size of the data is larger than the allowed token limited.  Each SK function
```

Comment:
I think the demo is not fully delivering on this unless the hardcoded limit of 2048 chars is removed from `GitHubSkill`

Reply:
SemanticTextPartitioner was added in, files are now indexed, regardless of their total size.